### PR TITLE
release notes for jan 8th 2023

### PR DESCRIPTION
**Pull request by allyford**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,42 @@
...
+
+### Release notes
+
+* Features
+  * Starting with Kubernetes 1.26:
+    * HostProcess Containers will be GA
+    * Azure NPM will now enforce ingress network policies for traffic sent via LoadBalancer IP's. The previous behavior NPM would only enforce pod to pod traffic. To avoid a breaking change, verify that you have CIDR block allow rules for your external traffic.
+    * Some AKS labels will be deprecated. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+  * [Dynamic allocation of IPs for Windows]( https://learn.microsoft.com/en-us/azure/aks/configure-azure-cni#:~:text=AKS%20cluster%20creation%3A-,Dynamic%20allocation%20of%20IPs%20and%20enhanced%20subnet%20support,-A%20drawback%20with) is now generally available
```
Comment (shashankbarsin) [inline CHANGELOG.md:19]:
```suggestion
  * [Dynamic allocation of IPs for Windows]( https://learn.microsoft.com/azure/aks/configure-azure-cni#:~:text=AKS%20cluster%20creation%3A-,Dynamic%20allocation%20of%20IPs%20and%20enhanced%20subnet%20support,-A%20drawback%20with) is now generally available
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+* Preview Features
+  * [IP based Load Balancer](https://learn.microsoft.com/en-us/azure/aks/load-balancer-standard#:~:text=Change%20the%20inbound%20pool%20type%20(PREVIEW)) is now available in preview
```
Comment (shashankbarsin) [inline CHANGELOG.md:21]:
```suggestion
  * [IP based Load Balancer](https://learn.microsoft.com/azure/aks/load-balancer-standard#:~:text=Change%20the%20inbound%20pool%20type%20(PREVIEW)) is now available in preview
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,42 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-01-08
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Kubernetes 1.25 has finished rolling out in all non-sovereign regions.
```
Comment (kaarthis) [inline CHANGELOG.md:9]: Perhaps call out why the delay in availability of 1.25 in some regions due to CCOA.

File: CHANGELOG.md (hunk 2 above)
Comment (shashankbarsin) [reply CHANGELOG.md:21]: same comment for all the other lines. we need to remove en-us to be localization friendly.

File: CHANGELOG.md (hunk 3 above)
Comment (phealy) [reply CHANGELOG.md:9]: CCOAs aren't to be publicly communicated, so I would just leave it as is.

File: CHANGELOG.md (hunk 1 above)
Comment (phealy) [inline CHANGELOG.md:19]:
```suggestion
  * [Dynamic Pod IP Allocation is now available for Windows nodes.]( https://learn.microsoft.com/en-us/azure/aks/configure-azure-cni#:~:text=AKS%20cluster%20creation%3A-,Dynamic%20allocation%20of%20IPs%20and%20enhanced%20subnet%20support,-A%20drawback%20with)
```

**Review by phealy — CHANGES_REQUESTED** [review]
Looks good pending Shashank's doc links and my wording about Dynamic Pod IP Assignment!

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,42 @@
...
+  * [Host port NSG control](https://learn.microsoft.com/en-us/azure/aks/use-node-public-ips) is now available in preview
+  * [Host ASG control](https://learn.microsoft.com/en-us/azure/aks/use-node-public-ips) is now available in preview
+  * [Azure CNI Overlay for Windows](https://learn.microsoft.com/en-us/azure/aks/azure-cni-overlay) is now available in preview
+* Behavior Changes
+  * The OSM addon's osm-injector pod's autoscaler will no longer scale on memory, only on CPU. The osm-controller pod's HPA resource will be reconciled in EnsureExists mode to allow you to edit the resource.
+* Bug Fixes
+  * A bug regarding Kubernetes version 1.25 and the tigera operator has been [fixed](https://github.com/Azure/AKS/issues/3394). If your cluster is already running on v1.25.4, please create a new cluster or file a support ticket with AKS for any further help.
+* Component Updates
+  * [CIS Kubernetes v1.24 Benchmark](https://learn.microsoft.com/en-us/azure/aks/cis-kubernetes) has been published which covers AKS 1.21.x through AKS 1.24.x
+  * KEDA add-on for AKS has been upgraded to v2.9
```
Comment (shashankbarsin) [inline CHANGELOG.md:32]: can we link to docs and release notes of v2.9 so that changelog is referenced

File: CHANGELOG.md (hunk 2 above)
Comment (allyford) [reply CHANGELOG.md:21]: Great catch! Didn't know about that - updating

File: CHANGELOG.md (hunk 4 above)
Comment (allyford) [reply CHANGELOG.md]: https://github.com/kedacore/keda/blob/main/CHANGELOG.md#v290 Should be this link, right?

**Review by shashankbarsin — APPROVED** [review]

**Review by qpetraroia — APPROVED** [review]

**Review by phealy — APPROVED** [review]
Approved, but please add the KEDA doc link per Shashank's comment before merging.

**Pull request merged**